kustomize/move2kube/overlays/prod/replace-config.yaml

```diff
@@ -0,0 +1,108 @@
+# replacement.yaml
+
+apiVersion: builtin
+kind: ReplacementTransformer
+metadata:
+  name: replace-config
+replacements:
+  - source:
+      kind: ConfigMap
+      name: move2kube-values
+      fieldPath: data.instance_name
+    targets:
```

Comment:
I am not familiar with replacement, so this will replaced all targets fields with the value of `data.instance_name`, correct?

Reply:
yes, it replaces the entire value of the field, whatever it is